LayoutTests/inspector/unit-tests/css-query-controller.html

```diff
@@ -72,12 +72,12 @@
                 specialIndices.forEach((index) => {
                     mask = mask.substr(0, index) + "^" + mask.substr(index + 1);
                 });
-                return mask.trim();
+                return mask;
```

Comment:
by using `trim` we lose the ability to know exactly where in the string the match(es) occurred, making it harder to find a regression

as an example, if today (using some metric) it is the case that the special indexes of the string `"abcdefghi"` were the first three characters (i.e. `"^^^______"`) then there'd be no way of knowing about a regression that suddenly caused the special indexes to become the last three characters (i.e. `"______^^^"`) because they'd both look identical after being `trim` (i.e. `"^^^"`)